db/migrate/20210712155135_add_section_to_navigation_links.rb

```diff
@@ -0,0 +1,5 @@
+class AddSectionToNavigationLinks < ActiveRecord::Migration[6.1]
+  def change
+    add_column :navigation_links, :section, :integer, default: 0, null: false
```

Comment:
With the addition of `null: false` here, can you run `rails db:migrate` again to get the change into `schema.rb`? 🙂